`measurements list` date error

If an empty date string is passed to the `--from` flag the program panics and errors:

```
panic: parsing time "" as "2006-01-02": cannot parse "" as "2006"

goroutine 1 [running]:
github.com/openaq/openaq-cli/cmd/measurements.glob..func2(0x14000197500?, {0x1400010ff20?, 0x4?, 0x100b8ae60?})
	github.com/openaq/openaq-cli/cmd/measurements/measurements.go:107 +0x214
github.com/spf13/cobra.(*Command).execute(0x100fa5b60, {0x1400010fec0, 0x3, 0x3})
	github.com/spf13/cobra@v1.7.0/command.go:944 +0x640
github.com/spf13/cobra.(*Command).ExecuteC(0x100fa3bc0)
	github.com/spf13/cobra@v1.7.0/command.go:1068 +0x320
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.7.0/command.go:992
github.com/openaq/openaq-cli/cmd.Execute()
	github.com/openaq/openaq-cli/cmd/root.go:37 +0x24
main.main()
	github.com/openaq/openaq-cli/main.go:9 +0x1c
```

This could be solved with a default value.

--- a/cmd/flags.go
+++ b/cmd/flags.go
@@ -52,7 +52,7 @@ func addPage(cmd *cobra.Command) {
 }
 
 func addFromDate(cmd *cobra.Command) {
-	cmd.PersistentFlags().StringVar(&FromDate, "from", "", "from")
+	cmd.PersistentFlags().StringVar(&FromDate, "from", "2000-01-01", "from")
 }
 
 func addToDate(cmd *cobra.Command) {

--- a/cmd/measurements/measurements.go
+++ b/cmd/measurements/measurements.go
@@ -47,12 +47,18 @@ func parseFlags(flags *pflag.FlagSet) (*openaq.MeasurementsArgs, error) {
 		measurementsArgs.Parameters = &parameters
 	}
 	from, err := flags.GetString("from")
+
 	if err != nil {
 		return nil, err
 	}
-	dateFrom, err := time.Parse("2006-01-02", from)
-	measurementsArgs.DatetimeFrom = dateFrom
-
+	var dateFrom time.Time
+	if from != "" {
+		dateFrom, err = time.Parse("2006-01-02", from)
+		if err != nil {
+			return nil, err
+		}
+		measurementsArgs.DatetimeFrom = dateFrom
+	}
 	if err != nil {
 		return nil, err
 	}
@@ -63,10 +69,10 @@ func parseFlags(flags *pflag.FlagSet) (*openaq.MeasurementsArgs, error) {
 	var dateTo time.Time
 	if to != "" {
 		dateTo, err = time.Parse("2006-01-02", to)
-		measurementsArgs.DatetimeTo = dateTo
 		if err != nil {
 			return nil, err
 		}
+		measurementsArgs.DatetimeTo = dateTo
 	}
 	return measurementsArgs, nil
 }

